File: src/main/java/client/database/data/AccountCashShopData.java
package client.database.data;

public class AccountCashShopData {
   private int nxCredit;

   private int maplePoint;

   private int nxPrepaid;

   public AccountCashShopData(int nxCredit, int maplePoint, int nxPrepaid) {
      this.nxCredit = nxCredit;
      this.maplePoint = maplePoint;
      this.nxPrepaid = nxPrepaid;
   }

   public int getNxCredit() {
      return nxCredit;
   }

   public int getMaplePoint() {
      return maplePoint;
   }

   public int getNxPrepaid() {
      return nxPrepaid;
   }
}

File: engine/model/src/main/scala/client/database/data/AccountCashShopData.scala
package client.database.data

class AccountCashShopData(private var _nxCredit: Int, private var _maplePoint: Int, private var _nxPrepaid: Int) {
  def nxCredit: Int = _nxCredit

  def maplePoint: Int = _maplePoint

  def nxPrepaid: Int = _nxPrepaid
}
